### ISSUE TITLE
export video

### ISSUE THREAD

**Issue by aster94** [nroduit/Weasis#115]
Hello, I wasn't able to find an easy way to convert a series of DICOM images into a video. I googled around and maybe it is possible with some plugin but there is not a list of available plugins so i am not sure where to find them

**Comment by xiaoluGood** [comment]
You can try Ffmpeg.

**Comment by aster94** [comment]
It doesn't look like it support it https://trac.ffmpeg.org/ticket/4728
By the way this is not a software recommendation but a feature request

**Comment by xiaoluGood** [comment]
FFmpeg is often used in my projects. If you want to export video, first you need to convert DICOM into JPG, and then use FFmpeg to synthesize MP4. In fact, the principle is JPG to synthesize video

**Comment by aster94** [comment]
ok, but wouldn't it be nice to have this already bundled inside weasis? 😄

**Comment by nroduit** [comment]
Yes, it should be possible to [write a video with OpenCV](https://docs.opencv.org/master/d7/d9e/tutorial_video_write.html) which the image lib used in Weasis.

Unfortunately my limited time on this project at the moment does not allow me to do this development as there are important changes underway to keep up with current technologies.

**Comment by brucejuz** [comment]
It is the only dicom viewer can work on my linux. Thanks!
I also hope weasis can conside to include the function which can convert the cine to video.
Thanks
